src/sentry_envelope.c

```diff
@@ -450,7 +454,7 @@ sentry__envelope_for_each_request(const sentry_envelope_t *envelope,
     sentry_uuid_t event_id = sentry__envelope_get_event_id(envelope);
 
     if (envelope->is_raw) {
-        req = prepare_http_request(&event_id, ENDPOINT_TYPE_STORE,
+        req = prepare_http_request(&event_id, ENDPOINT_TYPE_ENVELOPE,
             "application/x-sentry-envelope", envelope->contents.raw.payload,
```

Comment:
Would it be possible to introduce a constant for this?

Reply:
Will do when I refactor all of this ;-)